build/common.gypi

```diff
@@ -4,12 +4,14 @@
     'tizen_mobile%': 0,
     'shared_process_mode%': 0,
     'enable_murphy%': 0,
+    'disable_android_buildin_extensions%': 1,
```

Comment:
Set default as 0 please, this PR suppose back to master. We should control it in ./xwalk/gyp_xwalk